packages/design-system/src/new-ui/Tag/Tag.tsx

```diff
@@ -0,0 +1,65 @@
+import cn from "clsx";
+import React from "react";
+import { cva, type VariantProps } from "class-variance-authority";
+
+const tagVariants = cva(
+  "inline-flex items-center rounded-full border font-semibold transition-colors",
```

Comment:
@iamnamananand996 Because we don't have transition of color anymore, please remove `transition-colors`